test_suite/tests/test_macros.rs

```diff
@@ -812,6 +812,117 @@ fn test_internally_tagged_enum() {
     );
 }
 
+#[test]
+fn test_internally_tagged_enum_with_default_variant() {
```

Comment:
Is this the best way to test the macro?

I had a dig around the codebase, and I felt it made sense, but unsure.